lib/collection/variable-list.js

```diff
@@ -161,14 +219,16 @@ _.extend(VariableList, /** @lends VariableList */ {
      * @param {Object} items
      * @param {*} parent
      * @returns {PropertyList}
+     *
+     * @todo maybe move to instance methods or take instance as parameter? will help to create list specific keys and
+     * case senitivity
      */
     listify: function (items, parent) {
         // if item is plain object transform to array of objects
         _.isPlainObject(items) && (items = _.values(_.mapValues(items, function (value, key) {
-            return {
-                id: key,
-                value: value
-            };
+            var obj = { values: value };
```

Comment:
Earlier, we used to return an object something like this:
```
{ id: 'abc', value: 'def' }
```

With this change, we're now returning 
```
{ id: 'abc', values: 'def' }
```
I don't think this change is correct.